css/components/card-container.css

```diff
@@ -0,0 +1,11 @@
+.card-container{
+    background-color: var(--white);
+    width: 20rem;
+    height: 31.1875rem; 
```

Comment:
This is called "magic number", be cause it has lot of decimals that don't make sense, I mean, you have `499px` but with one more pixel you will have `500px` that is `31.25rem`, it is more clean than `31.1875`.
We usually work with maximum two decimals that almost always are `.25`, but there are some sumber like `14px` that are also used a lot even when it is `0.875rem`, but most of the time we use multiples of `4`, `8` or `16` so when you see a odd number, you have to rounded to a better value.
This does not happen with `box-shadows`, `border` and another properties, we use pixels for them, you will get to know them little by little.
Note: I said `border` not `border-radius`, we use `rem` for `border-radius`